mmv1/third_party/terraform/tests/resource_workstations_workstation_cluster_test.go.erb

```diff
@@ -45,13 +45,20 @@ func TestAccWorkstationsWorkstationCluster_update(t *testing.T) {
 
 func testAccWorkstationsWorkstationCluster_update(context map[string]interface{}) string {
 	return acctest.Nprintf(`
+data "google_project" "project" {
```

Comment:
Got the following error:
```
    vcr_utils.go:152: Step 3/4 error: Error running pre-apply refresh: 
        Error: Inconsistent dependency lock file
        
        The following dependency selections recorded in the lock file are
        inconsistent with the current configuration:
          - provider registry.terraform.io/hashicorp/google: required by this configuration but no version is selected
        
        To make the initial dependency selections that will initialize the dependency
        lock file, run:
          terraform init
```

This is probably due to `google_project.project` needing a `provider = google-beta` line

Reply:
I've added the provider argument in the data source.